Docs/Components/input.html

```diff
@@ -0,0 +1,219 @@
+                </a>
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="/Docs/Get-Started/typography.html">
+                    <div class="sidebar-item-txt">Typography</div>
+                </a>
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="/Docs/Get-Started/conclusion.html">
+                    <div class="sidebar-item-txt">Conclusion</div>
+                </a>
+
+                <div class="sidebar-heading">
+                    <h2 class="heading-2">Components</h2>
+                </div>
+                <hr class="divide-nav-items"/>
+
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./avatar.html">
+                    <div class="sidebar-item-txt">Avatar</div>
+                </a>
+    
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./alert.html">
+                    <div class="sidebar-item-txt">Alert</div>
+                </a>
+    
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./badge.html">
+                    <div class="sidebar-item-txt">Badge</div>
+                </a>
+
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./button.html">
+                    <div class="sidebar-item-txt">Button</div>
+                </a>
+    
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./card.html">
+                    <div class="sidebar-item-txt">Card</div>
+                </a>
+
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./image.html">
+                    <div class="sidebar-item-txt">Image</div>
+                </a>
+
+                <a class="sidebar-item centered-flex-row-container space-S rounded-med" href="./input.html">
+                    <div class="sidebar-item-txt link-active">Input</div>
+                </a>
+            </nav>
+        </aside>
+
+        <main class="component-container space-XL">
+            <section class="comp-intro">
+                <h1 class="heading-1 intro-heading">Input</h1>
+                <p class="intro-content para-4">
+                    Inputs are used to take data from the users, mostly by forms. The input element is the most important form element. The input element can be displayed in several ways, depending on the type attribute like 
+                    <span class="class-to-apply-for-comp">text</span>, <span class="class-to-apply-for-comp">radio</span>, <span class="class-to-apply-for-comp">checkbox</span> etc.
+                    You can also disabled them by using appropriate classes
+                </p>
+            </section>
+            <br>
+            <br>
+            <br>
+            <hr class="content-divider">
+            <br>
+            <br>
+            <br>
```

Comment:
Just a heads up, the `br` don't seem to be doing anything for the UI. It might be good to remove them and use margins for the `hr`

Reply:
Thanks for the suggestion! Will surely look into it.